notNeededPackages.json

```
@@ -8,6 +8,10 @@
             "libraryName": "hyphenate-style-name",
             "asOfVersion": "1.1.0"
         },
+        "@types/mock-property": {
+            "libraryName": "@types/mock-property",
```

```suggestion
        "mock-property": {
            "libraryName": "mock-property",
```

This is probably why things aren't working.

---

(It's on my list to try and fix the script to disallow this, there's at least one of these oopses checked in already)